daemon/cluster/cluster.go

```diff
@@ -51,6 +51,7 @@ import (
 	types "github.com/docker/docker/api/types/swarm"
 	executorpkg "github.com/docker/docker/daemon/cluster/executor"
 	"github.com/docker/docker/pkg/signal"
+	"github.com/docker/libnetwork/cluster"
```

Comment:
There is a package namespace collision `cluster`. Though it is not causing any direct harm, it will be good to call it something like `lncluster` and access the package using `lncluster` to make it more readable.